pkg/appgw/types.go

```diff
@@ -8,7 +8,7 @@ package appgw
 import (
 	"github.com/Azure/application-gateway-kubernetes-ingress/pkg/environment"
 	"github.com/knative/pkg/apis/istio/v1alpha3"
-	"k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
```

Comment:
You can omit `v1`

Reply:
That gets added automatically as v1 is imported in 2 imports.
```go
v1 "k8s.io/api/core/v1"
ptv1 "github.com/Azure/application-gateway-kubernetes-ingress/pkg/apis/azureingressprohibitedtarget/v1"
```